cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/ControlFlowSensitiveDFGPass.kt

```diff
@@ -45,12 +45,13 @@ import kotlin.contracts.contract
 @OptIn(ExperimentalContracts::class)
 @DependsOn(EvaluationOrderGraphPass::class)
 @DependsOn(DFGPass::class)
-open class ControlFlowSensitiveDFGPass(ctx: TranslationContext) : TranslationUnitPass(ctx) {
+open class ControlFlowSensitiveDFGPass(ctx: TranslationContext) : FunctionPass(ctx) {
```

Comment:
@KuechA should we do it for each resolution start holder instead?

Reply:
In the long term it should definitely work for each `ResolutionStartHolder` but we can filter for `FunctionDeclarations` for now.